src/components/markPosition.js

```diff
@@ -0,0 +1,11 @@
+export function markPosition(gameState, isFirstPlayerTurn, cell) {
+  const cellIndex = Array.from(cell.parentNode.children).indexOf(cell);
+  const chosenRow = Math.floor(cellIndex / 3);
+  const chosenPositionInRow = cellIndex - 3 * chosenRow;
+
+  if (isFirstPlayerTurn) {
+    gameState[chosenRow][chosenPositionInRow] = -1;
```

Comment:
Why `-1` and `1`? Using strings such as `circle` and `cross` would make it a lot easier to understand

Reply:
it was easier for me to check winning conditions with this type of data

Reply:
It's not very obvious which one is which. To deal with that, let's create a new file called `symbolsConstants.js`

```
export const circleValue = 1;
export const crossValue = -1;
```
and use them in places when you mean to refer to a cross or a circle.
```
if (isFirstPlayerTurn) {
  gameState[chosenRow][chosenPositionInRow] = crossValue;
} else {
  gameState[chosenRow][chosenPositionInRow] = circleValue;
}
```

Reply:
right, that's more clear - changed